docs/asciidoc/http-server.adoc

```diff
@@ -47,6 +47,20 @@ include::{examplesdir}/address/Application.java[lines=18..33]
 <2> Configures the `HTTP` server port
 ====
 
+To serve on multiple addresses, you can configure and bind to each address before configuring the next one. All created servers will share resources such as `LoopResources` because they use the same configuration instance under the hood.
```

Comment:
```suggestion
To serve on multiple addresses, after having configured the `HttpServer` you can bind it multiple times, obtaining separate `DisposableServer`s.
All created servers will share resources such as `LoopResources` because they use the same configuration instance under the hood.
```